### DISCUSSION TITLE
Update certain translation in tw language

### DISCUSSION THREAD

**Discussion by YourNameMitsuha** [lonqie/SchaleDB#69] [Translations]
The student's name in tw language should be 海花, not 海香.
![image](https://github.com/SchaleDB/SchaleDB/assets/105664116/ad899286-c500-4355-baa5-c8bc96cf2352)

In tw official translation, student's CV should keep its original japanese name insdead of translate to chinese.
The students that have CV issue are: **Ichika, Kasumi, Saten Ruiko, Yukari, Emi(Swimsuit), Kotama(Camping), Ibuki, Kayoko(Dress)**
![CompareImage](https://github.com/SchaleDB/SchaleDB/assets/105664116/58add4e7-244b-40a8-81e8-85a9426441fc)

And this should be translate to "以名稱篩選".
![螢幕擷取畫面 2024-03-29 223409](https://github.com/SchaleDB/SchaleDB/assets/105664116/dfb6149c-0069-4e94-9c2f-9f6130c15992)

**Comment by lonqie** [comment]
Moved this to discussions as it concerns translations and is not a website bug.

**Comment by factistel** [comment]
> The student's name in tw language should be 海花, not 海香. ![image](https://private-user-images.githubusercontent.com/105664116/318064874-ad899286-c500-4355-baa5-c8bc96cf2352.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3OTc4NzMsIm5iZiI6MTcxMTc5NzU3MywicGF0aCI6Ii8xMDU2NjQxMTYvMzE4MDY0ODc0LWFkODk5Mjg2LWM1MDAtNDM1NS1iYWE1LWM4YmM5NmNmMjM1Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMzMFQxMTE5MzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wNTNmZmYxNzRkY2Y3NzE2M2Y1NGM3ZjQ2ODE2ZTMxYWIxMTdkMjkwZWM4NmViMDZiM2EwZTJmYTg1NTkwZDNjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.p-o0hDHiM9r_YGfxqDm-FKzRnFjPYTR_CsZK7Ph3J4M)
> 
> In tw official translation, student's CV should keep its original japanese name insdead of translate to chinese. The students that have CV issue are: **Ichika, Kasumi, Saten Ruiko, Yukari, Emi(Swimsuit), Kotama(Camping), Ibuki, Kayoko(Dress)** ![CompareImage](https://private-user-images.githubusercontent.com/105664116/318068187-58add4e7-244b-40a8-81e8-85a9426441fc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3OTc4NzMsIm5iZiI6MTcxMTc5NzU3MywicGF0aCI6Ii8xMDU2NjQxMTYvMzE4MDY4MTg3LTU4YWRkNGU3LTI0NGItNDBhOC04MWU4LTg1YTk0MjY0NDFmYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMzMFQxMTE5MzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yODFmMzg0ODA3N2QyMzA5M2IzY2FhNWQzZWViOTJmODMwNmNjOTBkZDk1MjUxZWRkZGQxZDJiOWJmMWZiYmI1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9._8XNd_9blnI72Uq1HAngYXgNaRQb62lxeD6gCFnYnyg)
> 
> And this should be translate to "以名稱篩選". ![螢幕擷取畫面 2024-03-29 223409](https://private-user-images.githubusercontent.com/105664116/318070053-dfb6149c-0069-4e94-9c2f-9f6130c15992.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3OTc4NzMsIm5iZiI6MTcxMTc5NzU3MywicGF0aCI6Ii8xMDU2NjQxMTYvMzE4MDcwMDUzLWRmYjYxNDljLTAwNjktNGU5NC05YzJmLTlmNjEzMGMxNTk5Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMzMFQxMTE5MzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZjZjODJjNzg0NzY4ODE5NDQ1N2YzMWUxZWY5Y2Q4ZDk4ZDViOGRhYjQwOTU5M2QzZWZmYTY3NDMyZDU5MTA4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.TVU2h7xPF9nAqP3cDLYqqXsaiuadj804tXIlox9MyLk)

我就直接發中文了，說起來也方便些。

海花的名稱以及UI漏譯的文字已經修復了，感謝反饋。
關於CV名，我有如下觀點：
1. 能翻譯成中文的會傾向於翻譯成中文。（如果大部分遊戲的繁中都沒這麼做，即不翻譯CV名的話那當我這條沒說）
2. CV名的翻譯方針和國際服繁中的不同很正常，畢竟這裡提到的那部分是社群翻譯，也已經在首頁的特點裡說明過了（
3. 上面兩條在道理上是比不過這條的：一切以玩家為中心。也就是如果玩家社群接受哪種方案，那就遵循哪種翻譯方針。

關於第1點，我會再去查查資料（順帶確認下簡中和繁中在這方面的譯名上是否因文化而存在差異）。
但關於第2點，這裡就不得不吐槽一下一直以來就飽受詬病的（至少我個人覺得這樣）國際服翻譯。以技能名為例，拉丁文那些的技能名就不說了，畢竟韓文和日文都是直接把拉丁文擺上去，但是吧那些個日文裡帶片假名外來詞的，這繁中技能名就直接把片假名翻譯回外來詞，然後直接擺上去，就我個人而言，看到個叫`終幕：Ish-bosheth`（人名總不可能音譯不了吧）這樣的技能名，或者是好幾個技能名放在一塊，一眼望過去摻一點中文摻一點英文，而且看上去就完全能感到這部分摻進去的英文不是不能翻譯成中文，而是圖方便（或者根本不會英譯中）不翻譯，再加上那過去歷歷在目的各種繁中翻譯的豐功偉績，讓我很想找到那個翻譯繁中的傢伙噴個狗血淋頭。也因如此，我在某些方面對國際服繁中的信任度是很低的，這也許就是我選擇翻譯CV名（以及那些遊戲內術語）的原因之一吧。

如果真要選擇社群翻譯那部分的CV名和國際服繁中保持一致的話，還請在玩家社群中徵集足夠多的意見，這樣才好有充分的理由進行修改（那部分在國際服實裝後會換成國際服繁中的）

差不多就這些了

（在打這些字的時候還找到個文字有問題的地方，我這幾天還得仔細校對一遍）
